openlogin/login-with-github/index.lib.ts

```diff
@@ -0,0 +1,11 @@
+import * as playwright from "@playwright/test";
+
+export interface TestArgs {
+  openloginURL: string;
+  user: { email: string };
+}
+
+export const test = playwright.test.extend<TestArgs>({
+  openloginURL: "https://app.openlogin.com",
```

Comment:
Make this an env variable

Reply:
introduced an env variable PLATFROM which can be configured from github action file. possible values : PROD, STAGING